controllers/feeds/getFeedsV2.js

```diff
@@ -325,7 +325,7 @@ module.exports = async (req, res) => {
       if (data[i].own_reactions) {
         data[i].own_reactions = addKarmaScoreToLatestReaction(data[i].own_reactions, karmaScores);
       }
-      data[i] = modifyReactionsPost(data[i], data[i].anonimity);
+      data[i] = modifyReactionsPost(data[i]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyReactionsPost` function call has been updated to remove the `data[i].anonimity` parameter. However, this change could potentially introduce a breaking change if the `modifyReactionsPost` function expects two parameters. Please ensure that the function definition of `modifyReactionsPost` is updated accordingly and it can handle cases where only one parameter is passed.

```diff
- data[i] = modifyReactionsPost(data[i], data[i].anonimity);
+ data[i] = modifyReactionsPost(data[i]);
```

Also, please make sure that removing the `anonimity` parameter doesn't affect the functionality of the `modifyReactionsPost` function or any other parts of the codebase that rely on it.


<!-- This is an auto-generated comment by OSS CodeRabbit -->